packages/utils/src/libp2p/index.ts

```diff
@@ -89,12 +89,6 @@ export async function getConnectedPeersForProtocolAndShard(
 
     if (supportsProtocol) {
       if (shardInfo) {
-        //TODO: support auto-sharding
-        if (!("shards" in shardInfo)) {
-          throw new Error(
-            `Connections Manager only supports static sharding for now. Autosharding is not supported.`
-          );
-        }
```

Comment:
I'd expect a translation of autosharded params (if passed) to find the relevant peer since currently it only finds static sharded peers. would you agree?

Reply:
Cluster id is sufficient to find relevant peers for autosharding as well